schema/table.go

```diff
@@ -83,6 +83,20 @@ func (tt Tables) Get(name string) *Table {
 	return nil
 }
 
+// GetAll returns a table by name. Returns top-level tables and relations.
+func (tt Tables) GetAll(name string) *Table {
```

Comment:
I would make this the default `Get` and add a `GetTopLevel` etc. that looks only in the top level list.

Reply:
Maybe both `GetTopLevel` and `GetAll`?

Reply:
Keeping the `Get` name would keep compatibility across plugins, even though there's only a handful dest plugins (if not just PG) that's using it.

Reply:
`Get` (same as before) + `GetTopLevel` (new function) would get my vote

Reply:
Cool, I'm good with either. Let me update the PR